rdkit-postgresql95/meta.yaml

```diff
@@ -4,23 +4,27 @@ package:
 
 source:
   git_url: https://github.com/rdkit/rdkit.git
-  git_tag: Release_2016_09_2
+  git_rev: Release_2017_03
   patches:
-    - Makefile.patch
+    - cmakelists.txt.patch
     - adapter.cpp.patch
 
 build:
   number: 1
 
 requirements:
   build:
-    - python
-    - rdkit >=2016.09
-    - postgresql95
+    - cmake
+    - msys2-conda-epoch >=20160418 [win]
+    - m2-msys2-runtime [win]
+    - m2-libiconv [win]
+    - m2-libintl [win]
+    - m2-diffutils [win]
+    - m2-patch [win]
+    - postgresql >=9.5
```

Comment:
I think the only reason we "need" postgresql95 is to be able to use it from within the rdkit builds. Once the cartridge is build, I believe it can be installed in the standard conda postgresql 9.5 environment. This simplifies things for users who may want to install the RDKit cartridge along with some other package that has a postgresql dependency?

@ptosco : I think you also use the rdkit-postgresql* conda recipes. Do you have an opinion here?